src/udev/udevd.c

```diff
@@ -1456,6 +1457,13 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 
                 r = parse_sec(value, &arg_exec_delay_usec);
 
+        } else if (proc_cmdline_key_streq(key, "udev.timeout_signal")) {
+                if (proc_cmdline_value_missing(key, value))
+                        return 0;
+
+                r = signal_from_string(value);
+                if (r > 0)
+                        arg_timeout_signal = r;
```

Comment:
better: print warning here if we can't parse this

Reply:
This is handled couple of line below.

Reply:
Hmm...I meant this code bellow,

```C
        if (r < 0)
                log_warning_errno(r, "Failed to parse \"%s=%s\", ignoring: %m", key, value);

```

Reply:
Oh.